docs/training/apis-howto/api-core-ug.rst

```diff
@@ -1,212 +1,302 @@
 
-.. _core-metrics:
+.. note::
 
-****************
- Report Metrics
-****************
+   Throughout this user guide, we’ll show you how to make modifications to your script. After each
+   step, we’ll re-run our experiment using the appropriate script and its accompanying experiment
+   configuration file.
 
-The Core API makes it easy to report training and validation metrics to the master during training
-with only a few new lines of code.
+In this initial step, we’ll run our experiment using the ``model_def.py`` script and its
+accompanying ``const.yaml`` experiment configuration file.
 
-#. For this example, create a new training script called ``1_metrics.py`` by copying the
-   ``0_start.py`` script from :ref:`core-getting-started`.
+CD into the directory and run this command:
 
-#. Begin by importing import the ``determined`` module:
+.. code:: bash
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/1_metrics.py
-      :language: python
-      :start-after: NEW: import determined
-      :end-before: def main
+   det e create const.yaml . -f
 
-#. Enable ``logging``, using the ``det.LOG_FORMAT`` for logs. This enables useful log messages from
-   the ``determined`` library, and ``det.LOG_FORMAT`` enables filter-by-level in the WebUI.
+Open the Determined WebUI by navigating to ``http://localhost:8080/``. Accept the default determined
+username, leave the password empty, and click **Sign In**.
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/1_metrics.py
-      :language: python
-      :start-at: logging.basicConfig
-      :end-at: logging.error
+In the WebUI, select your experiment, and then navigate to the **Logs** tab.
 
-#. In your ``if __name__ == "__main__"`` block, wrap the entire execution of ``main()`` within the
-   scope of :meth:`determined.core.init`, which prepares resources for training and cleans them up
-   afterward. Add the ``core_context`` as a new argument to ``main()`` because the Core API is
-   accessed through the ``core_context`` object.
+************************
+ Step 2: Report Metrics
+************************
 
-   .. literalinclude:: ../../../examples/tutorials/core_api/1_metrics.py
-      :language: python
-      :start-at: with det.core.init
+To report training and validation metrics to the Determined master, we’ll add a few lines of code to
+our script. More specifically, we'll create a :class:`~determined.core.Context` object to allow
+interaction with the master. Then, we'll pass the ``core_context`` as an argument into ``main()``,
+``train()``, and ``test()`` and modify the function headers accordingly.
 
-#. Within ``main()``, add two calls: (1) report training metrics periodically during training and
-   (2) report validation metrics every time a validation runs.
+To run our experiment, we'll use the ``model_def_metrics.py`` script and its accompanying
+``metrics.yaml`` experiment configuration file.
```

Comment:
it does not appear clear that model_def_metrics.py is the result of applying the following changes to the previous file.

It reads more like the user should open that file and make the following changes.

Reply:
This appears in each section. I'll update each section to let the user know that the file they downloaded already has the changes in it.

To run the experiment, you can either use the files you downloaded, which have already been modified, or you can build your own file by making changes to "model_def.py".

Reply:
since there are other tutorials that use the python scripts and experiment configuration files, i don't want to rename them or have the user create their own files.

Reply:
resolved by adding a note to each major step